src/main/main.cpp

```diff
@@ -51,6 +51,11 @@ ultramodern::gfx_callbacks_t::gfx_data_t create_gfx() {
     SDL_SetHint(SDL_HINT_JOYSTICK_HIDAPI_PS5_RUMBLE, "1");
     SDL_SetHint(SDL_HINT_MOUSE_FOCUS_CLICKTHROUGH, "1");
     SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
+
+#if defined(__linux__) && defined(SDL_VIDEO_DRIVER_WAYLAND)
+    SDL_SetHint(SDL_HINT_VIDEODRIVER, "x11");
+#endif
```

Comment:
Why even check for wayland?
The hint should be harmless for x11 users
```suggestion
#if defined(__linux__)
    SDL_SetHint(SDL_HINT_VIDEODRIVER, "x11");
#endif
```

Reply:
I guess I wanted the check to be more precise, in case the user was using SDL2 built without Wayland support for some reason – but in reality it probably doesn't matter because the only thing that'll actually work is the X11 driver anyway.